.github/workflows/assets.yml

```diff
@@ -18,9 +18,10 @@ on:  # yamllint disable-line rule:truthy
       - Publish
     types:
       - completed
-    branches-ignore:
+    branches:
       - "[0-9]+.[0-9]+"
       - "[0-9]+.[0-9]+-stable"
+      - "[0-9]+.[0-9]+-lts"
```

Comment:
Are we planning to switch to -lts branch name? (Just curious).
FWIW we use X.Y.Z-lts tag names on the current X.Y-stable branch.

Reply:
The `make patch` over the stable branch creates a tag `tag: 8.12.4-lts` so I have added `-lts` for that reason since github action considers that as a branch when we refer the workflows. Not sure if the tags filter would work the way it should. Can have another PR for tags filter if needed